src/main.rs

```diff
@@ -1,10 +1,14 @@
 #![allow(unused_features)]
+#![feature(std_misc)]
 #![feature(static_mutex)]
 #![feature(box_syntax)]
+#![feature(libc)]
```

Comment:
But you did't include libc in Cargo.toml.  You need that to be using the crates.io one.